template.yaml

```diff
@@ -2390,6 +2390,12 @@ Resources:
                 bucket:
                   name:
                     - !Ref GroIngestionBucket
+      Logging:
+        Destinations:
+          - CloudWatchLogsLogGroup:
+              LogGroupArn: !GetAtt GroIngestionStateMachineLogGroup.Arn
+        IncludeExecutionData: true
```

Comment:
I think we want this to be false? To avoid logging all of the PII going through (again 🙃 )

Reply:
It seems to be only data about the state machine execution states (i.e., task state entered, task state exited). I've checked the logs and I can't find any PII so I think it should be okay?
<img width="318" alt="image" src="https://github.com/govuk-one-login/life-events-platform/assets/137415037/bba4c3c3-3a42-4b85-b114-9f53e130f291">

Reply:
Lol I forgot that we've implemented the removing PII from step functions already 🙃 if we hadn't it would be in the input/output, but since we've done that change this is fine